### ISSUE TITLE
Skeleton - at least one skeleton per example should include screen reader text

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#7290]
**Describe the enhancement or change**
This goes along with the a11y docs for the component. Basically a screen reader text element should be rendered inside of `pf-v6-c-skeleton` in one of the skeletons per example (to go along with the [when to apply screen reader text](https://www.patternfly.org/components/skeleton/accessibility#when-to-apply-screen-reader-text) verbiage from a11y docs (each example being a main section of the page).

One suggestion would be to utilize the example name, e.g. "Loading default example content".

**Any other information?**

**Comment by Mash707** [comment]
@thatblindgeye could you assign this to me.
